docs/image/image-security.md

```diff
@@ -0,0 +1,145 @@
+---
+id: image-security
+sidebar_position: 2
+sidebar_label: Image Security
+title: "Image Security"
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Image Security
+---
+
+_Available as of v1.4.0_
+
+Starting with v1.4.0 Harvester allows you to encrypt and decrypt virtual machine image. The encryption mechanism utilizes the Linux kernel module dm_crypt and the command-line utility cryptsetup.
+
+## Prerequisite
+
+Before encrypting or decrypting the virtual machine image, we need to prepare following resources:
+
+- source virtual machine image
+- secret
+- storage class
+
+### Source Virtual Machine Image
+
+Please follow [Upload Images](./upload-image) to create an image.
+
+:::info important 
+Harvester doesn't support uploading an encrypted image
+:::
+
+### Secret
+
+We use this secret as passphrase of dm_crypt. You need to customize the value of `CRYPTO_KEY_VALUE` field. Other fields are fixed.
+
+![](/img/v1.4/image/create-encryption-used-secret.png)
+
+Example Secret:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: encryption
+  namespace: default
+data:
+  CRYPTO_KEY_CIPHER: aes-xts-plain64
+  CRYPTO_KEY_HASH: sha256
+  CRYPTO_KEY_PROVIDER: secret
+  CRYPTO_KEY_SIZE: 256
+  CRYPTO_KEY_VALUE: "Your encryption passphrase"
+  CRYPTO_PBKDF: argon2i
+```
+
+:::info important
+
+If you'd like to create secret in the system namespace, you can use Edit as YAML or kubectl to create the secret. On the Harvester Secrets page, Harvester does not display resources in the system namespace.
+
+:::
+
+### Storage Class
+
+Since Harvesters uses Longhorn to encrypt image, we need to pass required fields to Longhorn CSI driver. You can select encryption used secret when creating storage class. If you're interested in this, please check [here](./upload-image#image-storageclass) for more details. 
+
+![](/img/v1.4/image/create-storage-class.png)
+
+Example Storage Class:
+
+```yaml
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: encryption
+parameters:
+  csi.storage.k8s.io/node-publish-secret-name: encryption
+  csi.storage.k8s.io/node-publish-secret-namespace: default
+  csi.storage.k8s.io/node-stage-secret-name: encryption
+  csi.storage.k8s.io/node-stage-secret-namespace: default
+  csi.storage.k8s.io/provisioner-secret-name: encryption
+  csi.storage.k8s.io/provisioner-secret-namespace: default
+  encrypted: "true"
+  migratable: "true"
+  numberOfReplicas: "3"
+  staleReplicaTimeout: "2880"
+provisioner: driver.longhorn.io
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+```
+
+:::info important
+
+If you'd like to select secret in the system namespace. You can use Edit as YAML or kubectl to create the secret. On the Harvester Secrets page, Harvester does not display resources in the system namespace.
+
+:::
+
+## Encrypt Image
+
+After getting a source image, please select `Ecnrypt` to encrypt virtual machine image.
```

Comment:
```suggestion
After getting an source image, please select `Ecnrypt` to encrypt virtual machine image.
```

Reply:
First character of `source` is not vowel, so it should be `a`, right?

Reply:
The first character of `source` is not vowel, it should be `a`, right?

Reply:
Sorry, I mean in L111. 
```
After getting a encrypted image,
```